plugins/woocommerce/src/Internal/DataStores/Orders/CustomOrdersTableController.php

```diff
@@ -282,39 +247,41 @@ private function get_settings_sections( array $sections ): array {
 
 	/**
 	 * Get the settings for the "Custom data stores" section in the "Advanced" tab,
-	 * with entries for managing the custom orders table if appropriate.
+	 * with entries for managing the custom orders tables if appropriate.
 	 *
 	 * @param array  $settings The original settings array.
 	 * @param string $section_id The settings section to get the settings for.
 	 * @return array The updated settings array.
 	 */
 	private function get_settings( array $settings, string $section_id ): array {
-		if ( ! $this->is_feature_visible() || 'custom_data_stores' !== $section_id || ! $this->data_synchronizer->check_orders_table_exists() ) {
+		if ( ! $this->is_feature_visible() || 'custom_data_stores' !== $section_id ) {
 			return $settings;
 		}
 
 		$title_item = array(
-			'title' => __( 'Custom orders table', 'woocommerce' ),
+			'title' => __( 'Custom orders tables', 'woocommerce' ),
 			'type'  => 'title',
 		);
 
-		$regeneration_is_in_progress = $this->data_synchronizer->data_regeneration_is_in_progress();
-
-		if ( $regeneration_is_in_progress ) {
-			$title_item['desc'] = __( 'These settings are not available while the orders table regeneration is in progress.', 'woocommerce' );
-		}
-
-		$settings[] = $title_item;
+		if ( $this->data_synchronizer->check_orders_table_exists() ) {
+			$settings[] = $title_item;
 
-		if ( ! $regeneration_is_in_progress ) {
 			$settings[] = array(
-				'title'         => __( 'Enable table usage', 'woocommerce' ),
-				'desc'          => __( 'Use the custom orders table as the main orders data store.', 'woocommerce' ),
+				'title'         => __( 'Enable tables usage', 'woocommerce' ),
+				'desc'          => __( 'Use the custom orders tables as the main orders data store.', 'woocommerce' ),
```

Comment:
During development I think we should add a warning here actually:
```suggestion
				'desc'          => __( 'WARNING: This feature is currently under development and will cause database instability. For contributors only.', 'woocommerce' ),
```

This would handle the case where a normal user would try to enable the feature on an old WooCommerce version.

Reply:
Ok, I have implemented it as an extra message on top of the setting instead of as a replacement of the original message.